@xen-orchestra/lite/src/components/PanelHeader.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { withState } from 'reaclette'
+
+import Icon, { IconName } from './Icon'
+
+import Button, {ButtonProps} from '@mui/material/Button';
```

Comment:
Could you run Prettier once to fix those? It shouldn't be necessary for future PRs on top of #5985.

Reply:
done